### PR TITLE
WT-12463 Remove connection->configure_method examples

### PR DESCRIPTION
**Pull request by korteland**
No tests for this one, just the two sections of examples. As far as I can tell, they only used the "call" side, and never used the configuration attached to the methods.

### CODE REVIEW

**Comment by korteland** [comment]
Update on this: the code won't build without the examples, since `s_all` expects certain references to be present. Do you think it's worth doing the documentation removal in this pull request?

File: dist/s_string.ok [hunk 1]
```diff
@@ -694,7 +694,6 @@ dmb
 dmsg
 drealloc
 dryrun
-ds
```
Comment (tod-johnson-mongodb) [inline dist/s_string.ok:697]: Question not about this change: How do you know what strings to delete? ./s_all tells you what strings to add. Does it also tell what strings to delete?

**Review by tod-johnson-mongodb — APPROVED** [review]
LGTM

File: dist/s_string.ok (hunk 1 above)
Comment (sueloverso) [reply dist/s_string.ok:697]: There is a `-r` flag to `s_string` that will build a replacement list of words. That gets periodically run to trim entries no longer needed as code gets removed.

**Comment by ruby-oujo** [comment]
Is it okay to remove this from `WT_DATA_SOURCE` ? Will it affect the custom data sources' API?

**Comment by korteland** [comment]
Closing this out - Ruby's observation about custom data sources is correct, and we're not removing those. So this needs to stay in.

**Comment by korteland** [comment]
Closing this out - Ruby's observation about custom data sources is correct, and we're not removing those. So this needs to stay in.

**Pull request closed without merging**